pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -629,10 +629,9 @@ func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, devi
 			resizeStatus, resizeErr := resizer.Resize(devicePath)
 
 			if resizeErr != nil {
-				resizeDetailedError := volumeToMount.GenerateErrorDetailed("MountVolume.resizeFileSystem failed", resizeErr)
+				resizeSimpleError, resizeDetailedError := volumeToMount.GenerateError("MountVolume.resizeFileSystem failed", resizeErr)
 				glog.Error(resizeDetailedError)
```

Comment:
do we need to log it either?

Reply:
Yeah, this log isn't necessary as `nestedPendingOperations.operationComplete` will finally log the error, I've removed it, PTAL.